core/modules/replica/Request.cc

```diff
@@ -150,6 +151,13 @@ string const& Request::remoteId() const {
 }
 
 
+unsigned int Request::nextTimeIvalMsec() {
+    auto result = _currentTimeIvalMsec;
+    _currentTimeIvalMsec = min(2 * _currentTimeIvalMsec, 1000 * timerIvalSec());
```

Comment:
some explanation about this calculation would be good IMO.

Reply:
It's in the header file in the Doxygen documentation for the method. I'm not sure it's safe to repeat it here in case if I'd gave to change the algorithm. Then I would make nods in 3 places instead of 2.